common/static/common/templates/discussion/new-post.underscore

```diff
@@ -9,20 +9,20 @@
     <% } %>
     <ul class="post-errors" style="display: none"></ul>
     <div class="forum-new-post-form-wrapper"></div>
-    <% if (cohort_options) { %>
+    <% if (group_options) { %>
```

Comment:
There seems to be a bug that this drop down is sometimes enabled when it should not be (when the selected thread is not divided). However, the bug appears to exist on master as well. I will investigate tomorrow.

![image](https://cloud.githubusercontent.com/assets/484484/25879120/ba2bb4ea-34fd-11e7-9f92-48478a95351b.png)